src/components/Person/Person.jsx

```diff
@@ -1 +1,11 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">I am {person.age}</p>}
+    <p className="Person__partner">
```

Comment:
Perhaps it will be better to move this logic above to improve maintainability and readability but your approach is also nice:
 
```
  const partnerMessage = person.isMarried
    ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
    : "I am not married";
    
    <p className="Person__partner">{partnerMessage}</p>
```